requirements.txt

```diff
@@ -1 +1,12 @@
-crhelper~=2.0.10
+-i https://pypi.org/simple
+boto3==1.24.40
+botocore==1.27.40; python_version >= '3.7'
+jmespath==1.0.1; python_version >= '3.7'
+numpy==1.23.1; python_version >= '3.8'
+pandas==1.4.3
+pyarrow==8.0.0
+python-dateutil==2.8.2; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
```

Comment:
I don't understand where `pipenv requirements` is getting these version constraints from when I explicitly have `python_version = "3.8"` in `Pipfile`, but I guess it's safe to remove them from the file since we know which version of python we'll be running on.